client-core/src/cipher/default.rs

```diff
@@ -29,7 +39,23 @@ fn get_tls_config() -> Arc<rustls::ClientConfig> {
     client_cfg.versions.clear();
     // TODO: test/try 1.3 and possibly switch
     client_cfg.versions.push(rustls::ProtocolVersion::TLSv1_2);
-    Arc::new(client_cfg)
+    Ok(Arc::new(client_cfg))
+}
+
+#[cfg(feature = "edp-obfuscation")]
+fn get_tls_config() -> Result<Arc<rustls::ClientConfig>> {
+    let verifier_config = EnclaveCertVerifierConfig {
+        signing_ca_cert_pem: IAS_CERT.into(),
+        valid_enclave_quote_statuses: vec!["OK".into(), "GROUP_OUT_OF_DATE".into()].into(),
```

Comment:
`GROUP_OUT_OF_DATE` isn't to be trusted

Reply:
Ok. I'll remove this. The reason I added this here is because during testing, I was not able to get `"OK"` status back. Maybe it is because I was using an old `SPID`?

Reply:
old Intel PSW (aesm etc.) or old CPU microcode / BIOS

Reply:
Changed